web/yo/app/scripts/directives/curationQueue.js

```diff
@@ -0,0 +1,354 @@
+                pre: function preLink(scope) {
+                    scope.queue = [];
+                    scope.data = {
+                        curatorsToNotify: [],
+                        curatorNotificationList: [],
+                        modifiedCurator: {}
+                    };
+                    scope.email = {
+                        status: {sending: false},
+                        returnMessage: ''
+                    };
+                    scope.queueModel = $rootScope.model.getRoot().get('queue');
+                    _.each(scope.queueModel.asArray(), function(item) {
+                        scope.queue.push({
+                            article: item.get('article'),
+                            pmid: item.get('pmid'),
+                            pmidString: 'PMID: ' + item.get('pmid'),
+                            link: item.get('link'),
+                            variant: item.get('variant'),
+                            addedBy: item.get('addedBy'),
+                            addedAt: item.get('addedAt'),
+                            curated: item.get('curated'),
+                            curator: item.get('curator')
+                        });
+                    });
+                    scope.getCuratorsList();
+                    scope.getArticleList();
+                    scope.dtOptions = {
+                        hasBootstrap: true,
+                        paging: false,
+                        scrollCollapse: true,
+                        aaSorting: [[0, 'asc']]
+                    };
+                    scope.dtColumns = [
+                        DTColumnDefBuilder.newColumnDef(0),
+                        DTColumnDefBuilder.newColumnDef(1),
+                        DTColumnDefBuilder.newColumnDef(2),
+                        DTColumnDefBuilder.newColumnDef(3).withOption('sType', 'date'),
+                        DTColumnDefBuilder.newColumnDef(4),
+                        DTColumnDefBuilder.newColumnDef(5),
+                        DTColumnDefBuilder.newColumnDef(6)
+                    ];
+                },
+                post: function postLink(scope) {
+                    scope.$watch('article', function(n, o) {
+                        if (n !== o) {
+                            $timeout.cancel(scope.articleTimeoutPromise);
+                            scope.articleTimeoutPromise = $timeout(function() {
+                                if (/^[\d]*$/.test(scope.article)) {
+                                    scope.getArticle(scope.article);
+                                }
+                            }, 500);
+                        }
+                    });
+                }
+            },
+            controller: function($scope) {
+                $scope.allCuration = false;
+                DatabaseConnector.getOncokbInfo(function(oncokbInfo) {
+                    if (oncokbInfo && oncokbInfo.users) {
+                        $scope.curators = oncokbInfo.users;
```

Comment:
Seems this is the first place declare `curators`, you should assign an empty array in `else`  or initialize the array in preLink